pkgs/development/python-modules/aws-secretsmanager-caching/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, botocore
+, codecov
+, pytest-cov
+, pytest-sugar
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "aws_secretsmanager_caching";
+  version = "1.1.1.5";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5cee2762bb89b72f3e5123feee8e45fbe44ffe163bfca08b28f27b2e2b7772e1";
+  };
+
+  propagatedBuildInputs = [
+    botocore
+  ];
+
+  nativeCheckInputs = [
+    pytest-cov
```

Comment:
We usually don't include coverage tests. Please remove `pytest-cov` and propably `codecov` from the dependencies and patch the Python build files to disable coverage tests